packages/angular-workspace/example-client-app/src/app/customapp/customapp.component.ts

```diff
@@ -217,12 +224,38 @@ export class CustomAppComponent implements AfterViewInit {
                 statusCode: (tableData.length % 2 === 0) ? 100 : 101,
                 result: possibleStatuses[tableData.length % 3],
                 number: tableData.length / 10,
-                duration: tableData.length * 1000 * (1.1 + 2 * 60 + 3 * 3600)
+                duration: tableData.length * 1000 * (1.1 + 2 * 60 + 3 * 3600),
+                color: this.possibleColors[tableData.length % 5]
```

Comment:
nit
```suggestion
                color: this.possibleColors[tableData.length % this.possibleColors.length]
```
(or `... % CustomAppComponent.possibleColors.length]` if you end up making it static.)